sdk/tables/Azure.Data.Tables/src/Extensions/DictionaryTableExtensions.cs

```diff
@@ -148,7 +148,7 @@ internal static T ToTableEntity<T>(this IDictionary<string, object> entity, Tabl
                 return result;
             }
 
-            typeInfo ??= TablesTypeBinder.Shared.GetBinderInfo(typeof(T));
+            typeInfo ??= TablesTypeBinder.Shared.GetBinderInfo(typeof(T), typeof(ITableEntity));
```

Comment:
Do we need to add this support to WebJobs extension?
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/PocoToTableEntityConverter.cs#L20

Reply:
Good callout!